spec/features/instant_sessions_spec.rb

```diff
@@ -1,21 +1,30 @@
 require "rails_helper"
 
 RSpec.describe "Instant sessions page" do
+  let(:api_response) { FactoryBot.build(:api_result, eligible: "eligible") }
+
   before do
-    visit root_path
-    visit "instant-controlled" # Trigger the session cookie to be set
+    visit root_path # Triggers the session cookie to be set
+
+    allow(CfeConnection).to receive(:assess).and_return(api_response)
   end
 
   scenario "I request an instant controlled session" do
     visit "instant-controlled"
     expect(page).to have_content "Check your answers"
+    expect(page).to have_content "Gross pay£1.00"
```

Comment:
I think I'm missing something - there isn't the text of "Gross pay" on the page only "Gross income", how are these bits passing?

Reply:
@MazOneTwoOne it's the joy of feature flags - in these specs, the self-employment FF isn't switched on, and that flag reworks all pay screens and how they're represented on the check answers screen.

(So when that feature flag is switched on everywhere and eventually deleted, we'll need to update this spec.)